code/modules/religion/sects/shadow_sect.dm

```diff
@@ -394,18 +375,36 @@
 	lisk.AddComponent(/datum/component/dark_favor, user)
 	lisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
 	playsound(altar_turf, 'sound/magic/fireball.ogg', 50, TRUE)
-	priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
-	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 100)
+	if(user.mind.is_murderbone())
+		priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	else
+		priority_announce("May our lord, [GLOB.deity], bless all the shadows as darkness heals those who worship the night.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate), user), 250)
 
-/obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate()
+/obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate(mob/living/user)
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	var/list/candidates = poll_ghost_candidates("Do you wish to be summoned as a Shadow Faithful?", ROLE_HOLY_SUMMONED, null, 10 SECONDS, POLL_IGNORE_HOLYUNDEAD)
 	for(var/obj/structure/destructible/religion/shadow_obelisk/obs in sect.obelisks)
 		START_PROCESSING(SSobj, obs)
+		if(!user.mind.is_murderbone())
+			playsound(obs, 'sound/magic/fireball.ogg', 50, TRUE)
+			return
```

Comment:
Should this be in the loop? I'd imagine it should come before anything else. Let me know if I'm mistaken.

Reply:
Good catch. this should store the result of the proc into a variable, then refer it to the local variable instead of calling the proc every time.

Reply:
Unsure how to have this run on each obelisk in sect.obelisks without the for loop, Might have done it before now but brain empty on the topic, if theres a way to have something run on everything in a list I can move those out of the loop easy enough

Reply:
`START_PROCESSING(SSobj, obs)` means you're going to trigger `/obj/structure/destructible/religion/shadow_obelisk/process(delta_time)`
in this loop, you only make it one, but don't do others because of `return`